src/server/status/server_status.js

```diff
@@ -2,15 +2,24 @@ import _ from 'lodash';
 
 import states from './states';
 import Status from './status';
+import PluginStatus from './plugin_status';
 
 module.exports = class ServerStatus {
   constructor(server) {
     this.server = server;
     this._created = {};
   }
 
-  create(plugin) {
-    return (this._created[plugin.id] = new Status(plugin, this.server));
+  create(id) {
+    const status = new Status(id, this.server);
```

Comment:
Similar to the note above re: renaming `create` -> `createForService`, should the `Status` class be renamed to `ServiceStatus` to mirror `PluginStatus`, for consistency?